server/src/test/java/com/linecorp/centraldogma/server/internal/storage/repository/cache/CachingRepositoryTest.java

```diff
@@ -69,17 +68,14 @@
 
 import io.micrometer.core.instrument.MeterRegistry;
 
-public class CachingRepositoryTest {
-
-    @Rule
-    public final MockitoRule mockito = MockitoJUnit.rule().strictness(Strictness.STRICT_STUBS);
+@ExtendWith(MockitoExtension.class)
```

Comment:
Could we register the `MockitoExtension` globally?
References: 
* https://github.com/mockito/mockito/pull/1391
* https://github.com/line/armeria/pull/1811
* https://github.com/line/armeria/blob/0a0767f5bf545cb05d97cf65f8a0b357376a8a9d/testing-internal/src/main/resources/META-INF/services/org.junit.jupiter.api.extension.Extension

Reply:
I had no idea this is possible, thanks for the suggestion!

Reply:
I had no idea this is possible, thanks for the suggestion!